agency/README.md

```diff
@@ -18,6 +19,18 @@ This specification contains a collection of RESTful APIs used to specify the dig
 * [Enum definitions](#enum-definitions)
 * [Responses](#responses)
 
+## General information
+
+The following information applies to all `agency` API endpoints. Details on providing authorization to endpoints is specified in the [Authorization](#authorization) section.
+
+### Versioning
+
+`agency` APIs must handle requests for specific versions of the specification from clients.
+
+Versioning must be implemented as specified in the [`Provider API versioning section`][Provider/versioning].
+
+Since versioning was not added until the 0.4.1 release, if the Accept header is not set as specified above, the Agency API must respond as if version 0.3 was requested.
```

Comment:
Alternatives would be :
1. Copy the content of the provider versioning section.
2. Create a separate `Versioning` file somewhere and reference it from the Provider, the Agency and the Policy specs.

Option 1 seems hard to maintain to me.
Option 2 would be a "cleaner" way of doing this but it requires to change the provider spec too.

Reply:
The same will then be done for the Policy spec, I just want to get to a consensus first.

Reply:
I like Option 2, seems like we might run into more of these use-cases and a `general-information.md` or similar file would be useful.